odbcabstraction/calendar_utils.cc

```diff
@@ -29,12 +30,7 @@ int64_t GetTodayTimeFromEpoch() {
 }
 
 void GetTimeForSecondsSinceEpoch(tm& date, int64_t value) {
-  #if defined(_WIN32)
-    gmtime_s(&date, &value);
-  #else
-    time_t time_value = static_cast<time_t>(value);
-    gmtime_r(&time_value, &date);
-  #endif
-  }
+  date = boost::posix_time::to_tm(boost::posix_time::from_time_t(value));
```

Comment:
There should be tests for SQL_DATE_STRUCT as well, not just SQL_TIMESTAMP_STRUCT

Reply:
Yep, planned, just getting timestamp sorted first.